clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -140,6 +138,15 @@ class ServerGroupController {
     return summaryList(application, cloudProvider)
   }
 
+  @PostAuthorize("@authorizationSupport.filterForAccounts(returnObject)")
+  @RequestMapping(value= "/serverGroups", method = RequestMethod.GET)
+  List getServerGroupsByApplications(@RequestParam(value = 'applications') List<String> applications,
+                                     @RequestParam(required = false, value = 'cloudProvider') String cloudProvider) {
+    def result = []
+    applications.forEach { application -> result +=summaryList(application, cloudProvider)}
```

Comment:
Could make the entire method body `applications.collectMany { app -> summaryList(application, cloudProvider) }`